MSCL/source/mscl/MicroStrain/MIP/Commands/MipCommand.cpp

```diff
@@ -102,8 +102,9 @@ namespace mscl
 
         /****   Read, Write     ****/
         // 0x0C
+        case MipTypes::CMD_EVENT_CONTROL:
         case MipTypes::CMD_GPIO_STATE:
-            return{
+            return {
```

Comment:
Really nitpicky, but it looks like most of the other lines don't have a space here

Reply:
Unfortunately spaces vs not is inconsistent in the code. In this particular function though spaces are there

Reply:
To go off that comment, this is something that was looked over when I did GPIO State and just noticed it was inconsistent in that function